packages/widgets-manager/src/grid.ts

```diff
@@ -67,10 +118,29 @@ export class GridManager {
 
   addAppToCell(id: string, app: App) {
     const existingCell = this._findCell(id);
```

Comment:
I think is better to add this `if (id.split(...)` and case inside the `_findCell` function, this way would be easier to refactor later, just an idea.

Reply:
Yep, the idea is to do properly in the next widget manager release, for that reason here I only did the split of the position segment. but yep in the end product, we need to do better :)